File: 2524-largest-positive-integer-that-exists-with-its-negative/largest-positive-integer-that-exists-with-its-negative.py
class Solution(object):
    def findMaxK(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max = -1
        for i in range(len(nums)):
            if nums[i] > max and -abs(nums[i]) in nums:
                max = nums[i]

        return max

File: 2524-largest-positive-integer-that-exists-with-its-negative/largest-positive-integer-that-exists-with-its-negative.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxK = function(nums) {
    let max = Math.max(... nums.filter((element) => {
        if(element > 0 && nums.includes(-element)){
            return element;
        }
    }));

    return max > 0 ? max : -1;
};